yolov5:
  action: False
  device: gpu
  img_size: 640
